pkgs/servers/home-assistant/default.nix

```
@@ -45,10 +45,14 @@ let
       });
     };
 
-  py = python3.override {
-    # Put packageOverrides at the start so they are applied after defaultOverrides
-    packageOverrides = lib.foldr lib.composeExtensions (self: super: { }) ([ packageOverrides ] ++ defaultOverrides);
-  };
+  py = python3.override(oa: {
+    # Put packageOverrides at the start so they are applied after
+    # defaultOverrides and upstream overrides
```

Maybe
```suggestion
    # defaultOverrides and overlay overrides
```
is a better wording?